src/Middleware/HeaderPropagation/src/HeaderPropagationEntry.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.AspNetCore.HeaderPropagation
+{
+    /// <summary>
+    /// Define the configuration of a header for the <see cref="HeaderPropagationMiddleware"/>.
+    /// </summary>
+
+    public class HeaderPropagationEntry
+    {
+        /// <summary>
+        /// Gets or sets the name of the header to be used by the <see cref="HeaderPropagationMessageHandler"/> fot the outbound http requests.
+        /// </summary>
+        public string OutboundHeaderName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the default values to be used when the header in the incoming request is missing or empty.
+        /// </summary>
+        public StringValues DefaultValues { get; set; }
```

Comment:
I noticied that this is `DefaultValueS` and the factory is `ValueFactory` (singular). What would you think about calling this `DefaultValue`?

Reply:
yes, it makes sense.